android/src/main/java/com/tadasr/IOTWifi/IOTWifiModule.java

```diff
@@ -123,6 +187,7 @@ public void removeSSID(String ssid) {
     public void getSSID(Callback callback) {
         WifiInfo info = wifiManager.getConnectionInfo();
         String ssid = info.getSSID();
+        Log.d("wifi", ssid);
```

Comment:
Add more to this log. Something like `Current wifi SSID: `

Reply:
Am just going to remove the logging statement.